homeassistant/components/ozw/__init__.py

```diff
@@ -114,7 +114,6 @@ def async_value_added(value):
         # Filter out CommandClasses we're definitely not interested in.
```

Comment:
Is there a good reason why we're not just importing all these CommandClasses on init?  I ran into issues due to `CommandClass.CONFIGURATION` being excluded in my PR causing tests to fail.

Reply:
This has recently changed. Please rebase this PR on latest dev branch and the filter for configuration will be gone.